### ISSUE TITLE
 Nested locator with first() 

### ISSUE THREAD

**Issue by TempleOfTemplar** [Codeception/CodeceptJS#2061]
#### What are you trying to achieve?
Get first element with attr tid="device" inside block with attr tid="requesting-devices-list", using following selector: locate("[tid=requesting-devices-list] [tid=device]").first()
#### What do you get instead?
Error: 
XPath with round brackets is not possible here! May be a nested locator with at() last() or first() causes this error.
> Provide console output if related. Use `--verbose` mode for more details.
 XPath with round brackets is not possible here! May be a nested locator with at() last() or first() causes this error.
```bash
# paste output here
```

> Provide test source code if related
locate("[tid=requesting-devices-list] [tid=device]").first()
```js
// paste test
```

### Details

* CodeceptJS version:
* NodeJS Version:
* Operating System:
* puppeteer || webdriverio || protractor || testcafe version (if related)
* Configuration file:

```js
# paste config here
```

**Comment by nicostubi** [comment]
This would be really useful to support nested locators

**Comment by esteban-serfe** [comment]
agreed. I try go generate .//tr[position()=1]/td/button[class="button"] and got this error

**Comment by kobenguyent** [comment]
Closed for now! Feel free to reopen if you still encounter the issue with latest version and would be nice to provide a sample code to reproduce the issue.
